Gemfile

```diff
@@ -2,12 +2,16 @@
 
 source 'https://rubygems.org'
 
-gem 'bcrypt_pbkdf', '>= 1.0', '< 2.0'
-gem 'ed25519', '>= 1.2', '< 2.0'
-gem 'net-sftp', '~> 2.1', '>= 2.1.2'
+gem 'rake'
 gem 'net-ssh'
+gem 'net-sftp'
 gem 'onlyoffice_digitalocean_wrapper'
-gem 'rake'
+
+# ed25519 adding support for net-ssh ed25219 key type library
+# bcrypt_pbkdf is used to encode\decode the received key
+# https://bugzilla.redhat.com/show_bug.cgi?id=1747751
+gem 'bcrypt_pbkdf', '>= 1.0', '< 2.0'
+gem 'ed25519', '>= 1.2', '< 2.0'
```

Comment:
I think there should be 2 comments, since gems are implemented different features, so if order of gems changed (I think in current state rubocop is not very happy) - different comments will help situation

Also not forgot to drop `>=1.2` since it's not really needed with `Gemfile.lock`